README.md

```diff
@@ -101,10 +101,25 @@ Some of the options available:
 
 ## invisible_recaptcha_tags
 
-[Invisible reCAPTCHA](https://developers.google.com/recaptcha/docs/invisible) has two differences:
+Make sure to read [Invisible reCAPTCHA](https://developers.google.com/recaptcha/docs/invisible).
 
-1. Needs a callback function, which is called after verification with Google's reCAPTCHA service. This callback function must submit the form.
-2. The `invisible_recaptcha_tags` generates a submit button.
+### With a single form on a page
+
+1. The `invisible_recaptcha_tags` generates a submit button for you.
+
+```Erb
+<%= form_for @foo do |f| %>
+  # ... other tags
+  <%= invisible_recaptcha_tags text: 'Submit form' %>
+<% end %>
+```
+
+Then, add `verify_recaptcha` to your controller as seen [above](#rails-installation).
+
+### With multiple forms on a page
+
+1. You will need a callback function, which is called after verification with Google's reCAPTCHA service. This callback function must submit the form.
```

Comment:
add something like this ?

build your own `invisibleRecaptchaSubmit` function and use `invisible_recaptcha_tags script: false`, see lib/recaptcha/client_helper.rb for details.

Reply:
I'm not sure why they need `script: false`? Based on this PR's implementation, they just need to pass in the name of their custom callback function and it will override the default callback.

So it would be more like: 
build your own invisibleRecaptchaSubmit function and use `invisible_recaptcha_tags callback: custom_callback` see lib/recaptcha/client_helper.rb for details.

Unless there was another reason for `script: false`?

Reply:
if someone implements `invisibleRecaptchaSubmit` and passes `callback: 'invisibleRecaptchaSubmit'` then the default script would be added again ... so I was thinking tell users to submit `script: false`

Reply:
Ah ok. Make sense. =)